### ISSUE TITLE
Nationality code standard

### ISSUE THREAD

**Issue by adamkcooper** [mosip/id-claim-169#17]
https://github.com/mosip/id-claim-169/blob/9b3e98795f17133e458e93f8ca54c199e818dcce/draft-id-claim-169.md?plain=1#L87

Does this follow ISO 3166-1 alpha-2? If so we should state this.

**Comment by gsasikumar** [comment]
Yes, it does.  You are right we will have to explicitly state it.  

@adamkcooper alpha-2 vs alpha-3 any preference?

**Comment by adamkcooper** [comment]
I would suggest alpha-2

**Comment by lsivanand** [comment]
@gsasikumar @adamkcooper How it will be handled incase of dual citizenship, where one can have two nationalities ?

**Comment by adamkcooper** [comment]
There is no need for more than one nationality to be recorded here. This is nationality as verified by the underlying ID system making the assertion.

**Comment by reshamchugani2** [comment]
**23rd Oct'25:**
1. It was recommended by the Claim 169 working group that MOSIP explores inclusion of a 2-letter ISO country code mapped to the 3-letter code, to accommodate international / ICAO requirements.

**Comment by reshamchugani2** [comment]
**4th Dec'25:**
- It was recommended to update the spec to support 2 letter code as per [ISO 3166-2 ](https://en.wikipedia.org/wiki/ISO_3166-2)standard and 3 letter code for ICAO support.
